src/components/SearchCard.tsx

```diff
@@ -75,23 +75,28 @@ export const SearchCard = forwardRef<HTMLDivElement, SearchCardProps>(
       >
         <Stack sx={{ ...responsiveStyle }}>
           <Stack
-            width={theme.spacing(sm || winWidth < 900 ? 16 : 36)}
-            height={theme.spacing(sm || winWidth < 900 ? 16 : 24)}
+            width={theme.spacing(mapCard || medium ? 16 : 36)}
+            height={theme.spacing(mapCard || medium ? 16 : 24)}
           >
             <ImageCarousel
-              size={sm || winWidth < 900 ? 'small' : 'large'}
+              size={mapCard || medium ? 'small' : 'large'}
               media={facility.media}
             />
           </Stack>
           <Stack
             onClick={() => navigate(`/facility/${facility.id}`)}
             justifyContent="space-between"
-            width={sm ? '252px' : '100%'}
-            spacing={1}
-            sx={{ py: 2, px: 1.5, mt: 0, cursor: 'pointer' }}
+            width={small ? theme.spacing(32) : '100%'}
```

Comment:
in the whole app, we handle responsive different then here, which makes it harder to read. Also we are creating exepcetions for naming, for example you use `small` but throughout the app we use the breakpoints `xs` or `sm`. 

You can either do this to make it responsive:

```const StackStyled = styled(Stack)(({ theme }) => ({
  width: theme.spacing(32),

  [theme.breakpoints.up('md')]: {
   width: "100%",
  }
}));```

Or this:

```
<Stack
  sx={{ width: { xs: theme.spacing(32), md: "100%"}}}

Reply:
outdated